src/Microsoft.AspNetCore.SignalR.Client.TS/HubConnection.ts

```diff
@@ -68,18 +67,16 @@ export class HubConnection {
     }
 
     invoke(methodName: string, ...args: any[]): Promise<any> {
-
-        let id = this.id;
```

Comment:
not sure about this change, won't the promise potentially capture the wrong value for id at https://github.com/aspnet/SignalR/pull/167/files#diff-98c1bfb809ef38f16f965a508fe52526R79